packages/ethers/src/utils/defaultConfig.ts

```diff
@@ -4,7 +4,6 @@ import { CoinbaseWalletSDK } from '@coinbase/wallet-sdk'
 
 export interface ConfigOptions {
   enableEIP6963?: boolean
-  enableInjected?: boolean
```

Comment:
I don't think we should add breaking changes by changing the interface. 
I'd say we leave the interface as-is but just not use the flag. 
We should probably emit a warning deprecation message instead.